sdks/java/core/src/test/java/org/apache/beam/sdk/runners/inprocess/InMemoryWatermarkManagerTest.java

```diff
@@ -816,18 +819,26 @@ public void synchronizedProcessingInputTimeIsHeldToUpstreamProcessingTimeTimers(
   }
 
   @Test
-  public void synchronizedProcessingInputTimeIsHeldToPendingBundleTimes() {
-    CommittedBundle<Integer> created = multiWindowedBundle(createdInts, 1, 2, 3);
-    manager.updateWatermarks(
-        null,
-        createdInts.getProducingTransformInternal(),
-        TimerUpdate.empty(),
-        Collections.<CommittedBundle<?>>singleton(created),
-        new Instant(29_919_235L));
+  public void synchronizedProcessingInputTimeIsHeldToPendingElementTimes() {
+    CommittedBundle<Integer> created = multiWindowedBundle(createdInts, 1, 2);
+    manager.updateWatermarks(null, createdInts.getProducingTransformInternal(), TimerUpdate.empty(),
+        Collections.<CommittedBundle<?>>singleton(created), new Instant(29_919_235L));
 
     Instant upstreamHold = new Instant(2048L);
+    WindowedValue<Integer> filteredElemOne =
+        WindowedValue.of(
+            1,
+            BoundedWindow.TIMESTAMP_MAX_VALUE.minus(100L),
+            ImmutableList.of(
+                GlobalWindow.INSTANCE,
+                new IntervalWindow(new Instant(0), BoundedWindow.TIMESTAMP_MAX_VALUE)),
+            PaneInfo.NO_FIRING);
     CommittedBundle<Integer> filteredBundle =
-        bundleFactory.createKeyedBundle(created, "key", filtered).commit(upstreamHold);
+        bundleFactory
+            .createKeyedBundle(created, "key", filtered)
+            .add(
```

Comment:
In the future please separate changes to style from changes to logic. It looks like your IDE changed a bunch of lines in ways that make it difficult to review/identify actual changes.